### DISCUSSION TITLE
ZuluSCSI Compact Homebrew Rev 2023b - how to connect CD-ROM eject switches?

### DISCUSSION THREAD

**Discussion by csulaguy** [ZuluSCSI/ZuluSCSI-firmware#319] [General]
I searched everywhere and cannot find any visual image of the Rev 2023b.  The board I received doesn't look at all like what was advertised on the sales site, so I'm wondering if this feature is now deprecated.

**Comment by aperezbios** [comment]
@csulaguy there are multiple images of the ZuluSCSI Compact Homebrew board on the product page, where you purchased it from, at [https://store.rabbitholecomputing.com/ZuluSCSI-Compact-Homebrew-p/zuluscsi-compact-homebrew-kit.htm](https://store.rabbitholecomputing.com/ZuluSCSI-Compact-Homebrew-p/zuluscsi-compact-homebrew-kit.htm)

Here's one of them
![image](https://github.com/ZuluSCSI/ZuluSCSI-firmware/assets/369925/d0716f9c-0fab-46bb-8344-c09d022a6d49)

I don't know if anyone has hooked up an eject button to the Compact Homebrew, but it should work, as the firmware is the same. You'd need to use the pins on J305. Ideally, this is the sort of thing that should have been confirmed _prior_ to purchase.

Here's a photo of the pinouts from the board layout ![image](https://github.com/ZuluSCSI/ZuluSCSI-firmware/assets/369925/f9679afa-c364-4827-b4c8-4ce43b9891da)
